### DISCUSSION TITLE
Cannot Login on own Hoppscotch Server

### DISCUSSION THREAD

**Discussion by sakhihartanto** [hoppscotch/hoppscotch#2885] [General]
Hi, can you guys tell me how to setup the Firebase .env on Docker ?
i run with
`docker run --rm --name hoppscotch -p 3000:3000 hoppscotch/hoppscotch:latest`
and i already change the .env with my own Firebase on Docker Exec in hoppscotch docker container but its still show this things.

![image](https://user-images.githubusercontent.com/61140070/208438835-62df8e89-38c5-48c0-a8b5-336c0acae1a6.png)

Can you guys help me to solve this ? or maybe can you show me how to config Firebase with step by step ?

**Comment by sebastianstucke87** [comment]
I just ran into the same problem, wondering why there seem to be no Docker ENVs in the documention (e. g. for a custom SMTP): see https://github.com/hoppscotch/hoppscotch/discussions/1923#discussioncomment-4371567
